src/languages/index.ts

```diff
@@ -30,3 +31,17 @@ export const TARGET_LANGUAGES: { [key in TargetLanguage]: VisitorFactory } = {
     createVisitor: () => new GoVisitor(),
   },
 };
+
+export function getVisitorFromLanguage(language: string | undefined) {
```

Comment:
Note that this function stays private because the files is not exposed:
https://github.com/aws/jsii-rosetta/blob/56836acc8f4a0f446686a67a3e0c1eef041ab55a/src/index.ts#L1-L14